bin/import-osm

```diff
@@ -20,36 +20,43 @@ function import_pbf() {
     local pbf_file="$1"
     local diff_flag=""
     if [ "$DIFF_MODE" = true ]; then
-        diff_flag="-diff"
```

Comment:
is this change necessary? Why is better to use the construction `${diff_flag:+"-diff"}`, instead of `$diff_flag` for the final command?

Reply:
This is not required, but it is a suggested solution for [SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086) -- look for `--trace-commands` example there.